.github/workflows/release-please.yaml

```diff
@@ -38,6 +38,11 @@ jobs:
                 "type":"build",
                 "section":"Build System or External Dependencies",
                 "hidden":false
+              },
+              {
+                "type":"expr",
+                "section":"Experimental",
```

Comment:
Or "Experimental Features" to make it clearer?

Reply:
I think we can do something like "exp(feat)", since not all exp changes necessarily have to be features. WDYT?